legal-api/src/legal_api/services/authz.py

```diff
@@ -486,9 +486,12 @@ def is_allowed(business: Business,
     is_ignore_draft_blockers = False
 
     if filing_id:
-        filing = Filing.find_by_id(filing_id)
-        if filing and filing.status == Filing.Status.DRAFT.value:
-            is_ignore_draft_blockers = True
+        if filing := Filing.find_by_id(filing_id):
+            if filing.status not in [Filing.Status.DRAFT.value, Filing.Status.CHANGE_REQUESTED.value]:
+                return False  # common for all filings
```

Comment:
until now resubmission fails while setting filing_json (have an additional validation in setter prop), which is in save_filing function.